Build fails in debug mode

**Describe the bug**
Building exiv2 0.27.1 fails when debug mode is enabled. This problem was [reported to MacPorts here](https://trac.macports.org/ticket/59932); a full build log is attached there.

**To Reproduce**
Steps to reproduce the behaviour:

Configure exiv2 with `-DDEBUG` in CFLAGS, CXXFLAGS; make as usual.

**Expected behavior**
Successful build.

**Desktop (please complete the following information):**
 - OS: macOS but that should be immaterial
 - Compiler & Version: AppleClang 10.0.0.10001044 in this case but that should be immaterial
 - Compilation mode and/or compiler flags: `-DDEBUG`

**Additional context**
The error encountered is:

```
src/pngimage.cpp:348:42: error: no matching function for call to 'binaryToString'
                            std::cerr << Exiv2::Internal::binaryToString(parsedBuf.pData_, parsedBuf.size_>50?50:parsedBuf.size_,0) << std::endl;
                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/image_int.hpp:125:36: note: candidate function template not viable: requires single argument 'sl', but 3 arguments were provided
    inline binaryToStringHelper<T> binaryToString(const Slice<T> sl) throw()
                                   ^
1 error generated.
```

This appears genuinely to be because `Exiv2::Internal::binaryToString` is being invoked with 3 parameters from pngimage.cpp when it is only defined with a single parameter in image_int.hpp.

I think the problem was caused by fec6535ae8a4299ea7f51606e45d3ea54b8e3194 which reimplemented `binaryToString` using slices, but neglected to make the necessary corresponding changes to this one call from pngimage.cpp. It is protected by an `#if DEBUG` preprocessor block which may explain how it was overlooked.

--- a/src/RemoteIo.cpp
+++ b/src/RemoteIo.cpp
@@ -29,6 +29,7 @@
 
 #include <cstring>  // std::memcpy
 #include <cassert>      /// \todo check usages of assert and try to cover the negative case with unit tests.
+#include <iostream>
 
 namespace Exiv2
 {

--- a/src/pngimage.cpp
+++ b/src/pngimage.cpp
@@ -352,9 +352,8 @@ namespace Exiv2
                         if (bExif || bIptc) {
                             DataBuf parsedBuf = PngChunk::readRawProfile(dataBuf, tEXt);
 #ifdef EXIV2_DEBUG_MESSAGES
-                            std::cerr << Exiv2::Internal::binaryToString(parsedBuf.pData_,
-                                                                         parsedBuf.size_ > 50 ? 50 : parsedBuf.size_, 0)
-                                      << std::endl;
+                            auto slice = makeSlice(parsedBuf.pData_, 0, parsedBuf.size_ > 50 ? 50 : parsedBuf.size_);
+                            std::cerr << Exiv2::Internal::binaryToString(slice) << std::endl;
 #endif
                             if (parsedBuf.size_) {
                                 if (bExif) {

